src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('error'));
+  }, 3000);
+});
+
+const successHandler = () => {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+};
+
+const errorHandler = (error) => {
+  const errormessage = document.createElement('div');
+
+  errormessage.classList.add('message', 'error-message');
+  errormessage.textContent = `Promise was rejected! ${error.message}`;
+  document.body.appendChild(errormessage);
+};
```

Comment:
This is a good example of an error handler function. However, you might want to consider making the error message more descriptive or user-friendly.